docs/hints-for-use-with-docker.md

```diff
@@ -57,3 +57,19 @@ To authenticate with a private Git repository, ORT uses the (semi)standardized `
    ```
 
    **Important:** Ensure that the `.netrc` file has been created at `netrc_folder_path` before running the command, otherwise Docker will create a folder `.netrc` inside the container, instead of mounting a single file.
+
+It is also possible to use `git config` to setup access to private repositories.
+
+1. Configure your gitconfig with your github credentials
```

Comment:
Instead of "gitconfig", this should say "`.gitconfig`" as below. Also "github" should be spelled "GitHub".

Reply:
Sorry, my quoting was probably not clear:

    `.github`

Should use backticks in the Markdown file.

Reply:
"gitconfig" is not written `.gitconfig`
"github" is now spelled "GitHub".